osf/models/user.py

```diff
@@ -814,15 +808,6 @@ def merge_user(self, user):
         # FOREIGN FIELDS
         self.external_accounts.add(*user.external_accounts.values_list('pk', flat=True))
 
-        # - addons
-        # Note: This must occur before the merged user is removed as a
-        #       contributor on the nodes, as an event hook is otherwise fired
-        #       which removes the credentials.
-        for addon in user.get_addons():
-            user_settings = self.get_or_add_addon(addon.config.short_name)
```

Comment:
@adlius Am I good to remove these lines? Publishing the changes via the signal `user_account_merged` should send a message to GV to merge the account's addons.

Reply:
They may be able to be removed if we indeed send a message to GV to merge those addons and GV does merged those addons upon receiving the message, which I am not sure.